src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -366,6 +363,40 @@ export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
                     />
                 </li>
             );
+            items.push(
+                <div
+                    className={classnames(
+                        'dropdown-item',
+                        styles.dropdownHoverEffect
+                    )}
+                    style={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        padding: '3px 20px',
+                    }}
+                >
+                    <form
+                        style={{
+                            display: 'flex',
+                            alignItems: 'center',
+                        }}
+                    >
+                        <input
+                            onClick={this.props.store.toggleNAValue(
+                                this.props.chartMeta.uniqueKey
+                            )}
+                            checked={this.props.store.isShowNAChecked(
+                                this.props.chartMeta.uniqueKey
+                            )}
+                            type="checkbox"
+                            style={{ margin: '0 5px 0 0' }}
+                        />
+                        <label style={{ margin: '0', fontWeight: 'unset' }}>
```

Comment:
may label clickable (See https://github.com/cBioPortal/cbioportal-frontend/blob/a66de296c183a3550dd759e02ef7ba7886e348da/src/pages/studyView/chartHeader/ChartHeader.tsx#L260)

Reply:
@kalletlak Luke found a bug: clicking on the checkbox seems to call the toggle function twice, even for the toggle log scale checkbox. I think maybe it's because the `a` tag that encapsulates the checkbox is also triggered when the checkbox is clicked.

Reply:
@jtquach1 is the issue fixed?

Reply:
It's fixed now, I just removed the `onClick` prop from the `FlexAlignedCheckbox`. Now the `onClick` is just on the surrounding `a` tag.